### PR TITLE
chore(skip-release): Prepare for 1.19.0

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by angelozerr** [comment]
@adietish there are so many changes in this release with the start stop of language server and lsp console.

Thats why it should be [1.19.0](https://github.com/redhat-developer/intellij-quarkus/milestone/30)

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -11,6 +11,13 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.19.0</b></p>
+    <ul>
+      <li>performance improvement which should avoid freezing IJ by improving start/stop of language server</li>
+      <li>LSP console to track LSP request/responses notification for all started language server</li>
```
Comment (fbricon) [inline src/main/resources/META-INF/plugin.xml:17]: language servers

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -11,6 +11,13 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.19.0</b></p>
+    <ul>
+      <li>performance improvement which should avoid freezing IJ by improving start/stop of language server</li>
```
Comment (fbricon) [inline src/main/resources/META-INF/plugin.xml:16]: servers

**Review by fbricon — CHANGES_REQUESTED** [review]

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml]: corrected

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml]: corrected

**Comment by adietish** [comment]
@fbricon, @angelozerr please re-review

**Comment by fbricon** [comment]
@adietish thanks, please proceed with the release

**Comment by adietish** [comment]
plugin 1.19.0.388 is available on jetbrains marketplace at https://plugins.jetbrains.com/plugin/13234-quarkus-tools/versions/stable/338515

**Pull request merged**